package.json

```diff
@@ -146,6 +146,9 @@
     "websocket": "1.0.28",
     "zkgroup": "https://github.com/signalapp/signal-zkgroup-node.git#2d7db946cc88492b65cc66e9aa9de0c9e664fd8d"
   },
+  "resolutions": {
```

Comment:
There's already a `resolutions` entry further down in this file. You'll need to move your entry there.